modules/xmpp/moderator.js

```diff
@@ -188,6 +188,29 @@ Moderator.prototype.createConferenceIq = function() {
             name: 'disableRtx',
             value: Boolean(this.options.conference.disableRtx)
         }).up();
+
+    if (this.options.conference.enableTcc !== undefined) {
+        elem.c(
+                'property', {
+                    name: 'enableTcc',
+                    value: Boolean(this.options.conference.enableTcc)
+                }).up();
+    }
+    if (this.options.conference.enableRemb !== undefined) {
+        elem.c(
+                'property', {
+                    name: 'enableRemb',
+                    value: Boolean(this.options.conference.enableRemb)
+                }).up();
+    }
+    if (this.options.conference.minParticipants !== undefined) {
+        elem.c(
+                'property', {
+                    name: 'minParticipants',
+                    value: this.options.conference.minParticipants
+                }).up();
```

Comment:
Are defaults set somewhere?

Reply:
The defaults are currently set in Jicofo. I documented this in the jitsi-meet config.js file, as per Pawel's suggestion. Let me know if you believe they should live elsewhere.